docs/cmd/jx-gitops_copy.md

```diff
@@ -0,0 +1,44 @@
+## jx-gitops copy
+
+Copies resources with the given selector or name from a source namespace to a destination namespace
+
+### Usage
+
+```
+jx-gitops copy
+```
+
+### Synopsis
+
+Copies kubernetes resources from a namespace to the current namespace
```

Comment:
```suggestion
Copies kubernetes resources (by default confimaps) from a namespace to the current namespace
```

Reply:
@jstrachan WDYT? Since ingresses can also be copied, saying the configmaps are the resources copied by default seems more clear.